Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4873,6 +4873,7 @@ def log_start_debug(self):
                                                                                             os.environ)
         if hasattr(demisto, 'params'):
             msg += "\n#### Params: {}.".format(json.dumps(demisto.params(), indent=2))
+        msg += "\n#### Args: {}.".format(json.dumps(demisto.args(), indent=2))
```

Comment:
i might be missing something, but i think we already print the command args at the beginning of the log

Reply:
We don't. I encountered a log where I needed this and couldn't find it.

Reply:
isnt done in https://github.com/demisto/server/blob/master/services/automation/dockercoderunner.go#L718 ?

Reply:
You are correct. Not sure why I didn't see this. I'll remove this anyway until I encounter this again as I can't find the log I saw this on.